src/wp-admin/includes/class-wp-plugins-list-table.php

```diff
@@ -759,6 +759,54 @@ public function single_row( $item ) {
 		$compatible_php = is_php_version_compatible( $requires_php );
 		$compatible_wp  = is_wp_version_compatible( $requires_wp );
 
+		if ( ! function_exists( 'plugins_api' ) ) {
+			require_once ABSPATH . 'wp-admin/includes/plugin-install.php';
+		}
+
+		$plugin_information = plugins_api(
+			'plugin_information',
+			array(
+				'slug'   => $plugin_slug,
+				'fields' => array(
+					'sections' => false,
+					'rating'   => false,
+					'downloaded' => false,
+					'last_updated' => false,
+					'banners' => false,
+					'icons' => false,
+					'active_installs' => false,
+					'short_description' => false,
+					'tags' => false,
+					'compatibility' => false,
+					'contributors' => false,
+					'ratings' => false,
+					'screenshots' => false,
+					'support_url' => false,
+					'added' => false,
+					'homepage' => false,
+					'download_link' => false,
+					'upgrade_notice' => false,
+					'versions' => false,
+					'business_model' => false,
+					'repository_url' => false,
+					'commercial_support_url' => false,
+					'donate_link' => false,
+					'preview_link' => false,
+				),
```

Comment:
Better to use `array_fill_keys()` to reduce duplication:

```suggestion
				'fields' => array_fill_keys(
					array(
						'sections',
						'rating',
						'downloaded',
						'last_updated',
						'banners',
						'icons',
						'active_installs',
						'short_description',
						'tags',
						'compatibility',
						'contributors',
						'ratings',
						'screenshots',
						'support_url',
						'added',
						'homepage',
						'download_link',
						'upgrade_notice',
						'versions',
						'business_model',
						'repository_url',
						'commercial_support_url',
						'donate_link',
						'preview_link',
					),
					false
				),
```